web/impact/impact/tests/test_algolia_api_key_view.py

```diff
@@ -101,18 +101,18 @@ def test_entrepreneur_that_never_had_a_finalist_role_gets_403(self):
             self.assertTrue(response.status_code, 403)
 
     def test_user_with_staff_role_grant_sees_all_mentors(self):
-        user = self.basic_user()
+        user = self.staff_user()
         program = _create_batch_program_and_named_group(
                         ACTIVE_PROGRAM_STATUS, 1)
-        self._create_user_with_role_grant(program[0], UserRole.STAFF, user)
+        self._create_user_with_role_grant(program[0], user)
```

Comment:
I don't think we need this anymore

Reply:
Am on it

Reply:
@VincentHokie @shakirandagire Just to be clear on this, we are maintaining the Staff ProgramRoleGrant for the time being, as it is important for (among other things) seeing navigation on the accelerate side. 
I'm not aware of any place where we use this on the impact side, so the question is whether we care more about symmetry of "staff" status across the two repos, or simplicity on the impact side. 
I'm okay with these changes as-is, I just wanted to make sure we were all on the same page about this.

Reply:
yup, we're on the same page @jonkiparsky . thanks for the context..

Reply:
@jonkiparsky I think the symmetry of staff status across the two repos would be ideal. And as @VincentHokie has stated, we are on the same page.

Reply:
Great, thanks. So if you want to restore the staff Role Grants to maintain symmetry, I can hold off on merging until that's done, or I can merge now. Let me know which you'd prefer.